ag_data/formulas/tests.py

```diff
@@ -66,6 +82,36 @@ def test_save_measurement_identity(self):
         self.assertEqual(result.event_uuid, self.event)
         self.assertEqual(result.sensor_id, sensor)
 
+    def test_no_event(self):
```

Comment:
I think we should be checking these things here: a) it will raise an assertion error b) it will write a log into the database c) the log is in correct shape and format.

Reply:
I have added an assert for that exception, for b) and c), it is already achieved.